### PR TITLE
escape the dots for the regexen

### PR DESCRIPTION
**Pull request by gabrielsroka**
## Description:
i'm guessing that for these regexen, instead of:
`/example.okta/`
you wanted:
`/example\.okta/`

also, you don't need to escape `'/'`

### CODE REVIEW

File: scripts/markdown-lint.js [hunk 1]
```diff
@@ -23,14 +23,14 @@ const regexValidator = [
   // (Optional) - Pass in a list of files that can be ignored from the check.
   // For global ignores, update the getFiles() method.
   {
-    regex: 'https?:\/\/(your-org|example|rain|your-subdomain|your-domain|{org}).okta*',
+    regex: 'https?://(your-org|example|rain|your-subdomain|your-domain|{org})\\.okta*',
```
Comment (jmelberg-okta) [inline scripts/markdown-lint.js]: The trailing `\\` aren't required for this regex, so they can safely be removed.

File: scripts/markdown-lint.js [hunk 2]
```diff
@@ -23,14 +23,14 @@ const regexValidator = [
...
     omitFiles: [
       '/_docs/api/postman/apps.json',
       '/_docs/api/postman/example.oktapreview.com.environment'
     ]
   },
   {
-    regex: 'https?:\/\/{yourOktaDomain}.com*',
+    regex: 'https?://{yourOktaDomain}\\.com*',
```
Comment (jmelberg-okta) [inline scripts/markdown-lint.js]: Same as above

**Comment by gabrielsroka** [comment]
> ESLint

not a bad idea. will it know that `str = '.com'` which will later be used in `RegExp(str)` needs to be escaped? if so, wow! or will it just catch many other errors?

**Comment by nbarbettini** [comment]
@gabrielsroka Can you incorporate Jordan's feedback? Then we can merge this.

File: scripts/markdown-lint.js (hunk 1 above)
Comment (gabrielsroka) [reply scripts/markdown-lint.js]: @jmelberg-okta 
the `\\.` is in a string which becomes this regex : `/\./`. if you take them out, the regex is too loose: `/./` will match any character -- we want a literal dot.

File: scripts/markdown-lint.js (hunk 2 above)
Comment (gabrielsroka) [reply scripts/markdown-lint.js]: @jmelberg-okta same as above

**Comment by bretterer** [comment]
Thank you for this PR to help improve our documentation. As you may know, we have moved our developer documentation to a new repo.

Since this PR is targeting our developer docs, we are requesting that you migrate this PR over to the new repo https://github.com/okta/okta-developer-docs

If you have any questions about this, or need assistance with a complex PR, please let us know in #eng-devex

**Pull request closed without merging**